MyQuizAdmin/MyQuizAdmin/App.xaml.cs

```diff
@@ -14,16 +15,12 @@ namespace MyQuizAdmin
     /// </summary>
     sealed partial class App : Application
     {
+        public static ObservableCollection<string> groups = new ObservableCollection<string>();
```

Comment:
Why is this an observable collection of strings?
Shouldn't this be a collection of groups?